src/controllers/TripsController.js

```diff
@@ -0,0 +1,65 @@
+import Responses from '../utils/response';
+import db from '../models';
+/**
+ * Class for Trips
+ */
+const TripsController = {
```

Comment:
we need to decide on whether to use classes or this particular pattern, different implementation patterns in the same codebase brings confusion.

Reply:
You are right! we need to discuss this. because classes bringing may issues